lib/Dialect/FIRRTL/Export/FIREmitter.cpp

```diff
@@ -253,6 +253,18 @@ struct Emitter {
                                  [&](Value v) { emitSubExprIBox2(v); });
   }
 
+  /// Emit a (potentially nested) symbol reference as `A.B.C`.
+  void emitSymbol(SymbolRefAttr symbol) {
+    ps.ibox(2, IndentStyle::Block);
```

Comment:
FWIW to help ensure boxes are closed and show their scoping there's `ps.scopedBox(PP::ibox2, [&]() { ... })` pattern.  This helps in more complicated code esp when there are return paths to "cover" and various nested boxes.  But this is fine as-is, just FYI! :)